### PR TITLE
Drop parentheses around selected DB fields

### PR DESCRIPTION
**Pull request by carsonb**

### CODE REVIEW

**Review by iamkevb — APPROVED** [review]

**Comment by carsonb** [comment]
⚠️ I added a library to handle `camelCase`, hope you don't mind

File: ast/ast.go [hunk 1]
```diff
@@ -76,6 +76,14 @@ func (m *Model) FieldSlice() []string {
...
 }
 
+func (m *Model) FieldSliceInCamelCase() []string {
+	out := []string{}
+	for _, v := range m.Fields {
+		out = append(out, v.NameInCamelCase())
+	}
+	return out
+}
+
```
Comment (iamkevb) [inline ast/ast.go:86]: You really only need to lowerCase the first character. This code will do that without the library:
```go
out = append(out, strings.ToLower(v[:1])+v[1:])
```

Comment (carsonb) [reply ast/ast.go:86]: Okay. I picked the full camelCase to as a sledgehammer to fix the situation where `ID` would become `id`, instead of `iD`

Comment (iamkevb) [reply ast/ast.go:86]: ah.. `id` is nicer that `iD`. @boourns, you cool with the library?

Comment (boourns) [reply ast/ast.go:86]: Yep works for me

**Review by boourns — APPROVED** [review]
Thanks!

File: ast/ast.go (hunk 1 above)
Comment (carsonb) [reply ast/ast.go:86]: <3 
This morning, I considered a way to handle those cases...and it basically becomes implementing what's in the library.

**Pull request merged**